src/main/java/io/github/railroad/objects/CreateNewJavaFile.java

```diff
@@ -33,18 +33,19 @@ public void fileDialogBox(Stage window) {
 
 	}
 
+	//TODO make it open the file in editor after saving
 	@Override
 	protected Button saveFile(Stage window) {
 		Button yesBtn = UIUtils.createButton(this.message, event -> {
 			File file = FileUtils.createNewFile(filePath);
 			String code = "";
-			if (this.type == 1) {
+			if (this.type == JavaClassTypes.CLASS.ID) {
```

Comment:
You could make the JavaClassTypes an enum.
Also maybe use switch statements to make it look cleaner.

Reply:
It is an enum?

Reply:
sorry yeah, it is an enum. But instead of making this.type and Integer, you should make it a JavaClassTypes.
Also, ID is basically already implemented in enums. Enums have a method called Enum#ordinal() which basically reuturns its index.